src/event.rs

```diff
@@ -177,6 +181,39 @@ impl<'a, N: Notify + 'a> input::ActionContext for ActionContext<'a, N> {
     fn clear_log(&mut self) {
         self.terminal.clear_log();
     }
+
+    fn new_instance_same_dir(&mut self) {
+        #[cfg(target_os = "linux")]
+        unsafe {
+            let current_working_dir = Command::new("pwdx")
```

Comment:
I think you don't need `pwdx`. try [cwd](https://github.com/danburkert/procinfo-rs/blob/master/src/pid/cwd.rs#L12).

Reply:
It works. :fire: